rpcpb/rpc.proto

```diff
@@ -331,6 +337,11 @@ message RestartNodeRequest {
   // If specified, will create a file "chainname/upgrade.json" with
   // the contents provided here.
   map<string, string> upgrade_configs = 5;
+
+  // Map of subnet id to subnet config file contents.
+  // If specified, will create a file "subnetid.json" under subnets config dir with
+  // the contents provided here.
+  map<string, string> subnet_configs = 6;
```

Comment:
Shouldn't we actually set `optional` to all these configs elements?

Reply:
optional makes the value to be nil if not given, which is already the behaviour for maps

Reply:
OK but to me it's makes things explicit for every one else. I did not know for example  that the value is nil if not given.
 
So I think it's clearer to set it optional where it needs to be optional

Reply:
also the change should be applied to all map params in RPC so another PR seems best